### PR TITLE
Graphql sectors resolvers

### PR DESCRIPTION
**Pull request by IdezHD**
- Created sector resolver's
- Created tests for resolver's

### CODE REVIEW

File: api/tests/test_sector_resolver.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+			getSectorsBySector(sector: "Should Not Work") {
+				id
+				zone
+				description
+			}
+		}
+		"""
+		executed = client.execute(query)
+		assert executed['errors']
+		assert executed['errors'][0]
```
Comment (sleepycat) [inline api/tests/test_sector_resolver.py]: Not sure this particular assertion is a big "value add".

File: api/queries.py [hunk 2]
```diff
@@ -1,11 +1,48 @@
...
 	all_users = SQLAlchemyConnectionField(UserConnection, sort=None)
+	# all_users = graphene.List(UserObject, failedAttempts=graphene.Int(), resolver=resolve_all_users)
+	get_sector_by_id = graphene.List(
+		of_type=Sectors,
+		id=graphene.Argument(graphene.Int, required=False),
+		resolver=resolve_get_sector_by_id
+	)
+	get_sectors_by_sector = graphene.List(
+		of_type=Sectors,
+		sector=graphene.Argument(graphene.String, required=False),
```
[the remark this answers is not in the document]
Comment (IdezHD) [reply api/queries.py]: There are currently 24 different sectors that we have in the db

File: api/queries.py [hunk 3]
```diff
@@ -1,11 +1,48 @@
...
+		resolver=resolve_get_sectors_by_sector
+	)
+	get_sector_by_zone = graphene.List(
+		of_type=Sectors,
+		zone=graphene.Argument(graphene.String, required=False),
```
[the remark this answers is not in the document]
Comment (IdezHD) [reply api/queries.py]: We do have only 2 zones currently, but that could change, would enums be best for that?

**Comment by IdezHD** [comment]
Found bug that breaks flask-migrate **DO NOT MERGE**

**Pull request merged**